fpo-api/api/send_email.py

```diff
@@ -8,30 +9,34 @@
 from email.mime.text import MIMEText
 from email.header import Header
 from email.utils import formataddr
+import requests
+import base64
 
 from smtplib import SMTP, SMTPException
 
 LOGGER = logging.getLogger(__name__)
 
-
-def send_email(recipient_email: str, pdf_data: bytes, pdf_name: str):
-    server_addr = settings.SMTP_SERVER_ADDRESS
+def send_email(recipient_email: str, pdf_data: bytes, pdf_name: str, auth_token: str)->{}:
     sender_email = settings.SMTP_SENDER_EMAIL
-    sender_name = settings.SMTP_SENDER_NAME
-    if not server_addr:
-        LOGGER.debug("SMTP server address not configured")
-        return
+    #sender_name = settings.SMTP_SENDER_NAME
+
     if not sender_email:
         LOGGER.error("Sender email address not configured")
         return
-    if not sender_name:
-        LOGGER.error("Sender name not configured")
-        return
     if not recipient_email:
         LOGGER.error("No recipient email address provided")
         return
 
-    subject = "Traffic Hearing Choice"
+    # Update filename 
```

Comment:
This pdf file naming logic appears in a more than one location.  The name of the file should be determined once when first generated, saved to the database in the `TicketResponse.pdf_filename` field and then passed around from there.